pulpsolv.py

```diff
@@ -327,16 +327,39 @@ def __call__(self, unit):
         return solvable
 
 
+class SrpmSolvableFactory(BasetUnitSolvableFactory):
+    # An SRPM content unit factory
+    attribute_factories = [
+        AttributeFactory('name'),
+        EVRAttributeFactory(),
+        # FIXME: any other arch makes sense?
+        AttributeFactory('arch', conversion=lambda x: 'noarch'),
```

Comment:
FIXME**2 works the moment I set arch to noarch... I don't mind but I sorta hope there's a better way O:-)